assets/src/edit-story/elements/video/karma/autoplay.karma.js

```diff
@@ -68,5 +73,26 @@ describe('Autoplay video', () => {
     const backgroundId = await getBackgroundElementId(fixture);
     const backgroundElVideo = fixture.querySelector(`#video-${backgroundId}`);
     expect(backgroundElVideo.paused).toBe(false);
+
+    // Bug #2618 coverage
+    const backgroundEl = fixture.querySelector(
+      `[data-testid="safezone"] [data-element-id="${backgroundId}"]`
+    );
+    await fixture.events.mouse.seq(({ moveRel }) => [
+      moveRel(backgroundEl, '10%', '10%'),
+    ]);
+    await waitFor(() =>
+      fixture.screen.getByRole('button', { name: 'Click to pause' })
+    );
+    const pauseButton = fixture.screen.getByRole('button', {
+      name: 'Click to pause',
+    });
+    await fixture.events.click(pauseButton);
+    expect(backgroundElVideo.paused).toBe(true);
+    const playButton = fixture.screen.getByRole('button', {
+      name: 'Click to play',
+    });
+    await fixture.events.click(playButton);
+    expect(backgroundElVideo.paused).toBe(false);
```

Comment:
Could we rewrite this (entire test) to use [the "fake DOM" container structure](https://github.com/google/web-stories-wp/blob/fix/video-play-edge-cases/assets/src/edit-story/karma/fixture/containers/editor.js#L29)? And add the remaining properties and containers there if missing?

Reply:
Also can we wrap it in [a proper CUJ for this](https://github.com/google/web-stories-wp/issues/1587) - probably something akin to `describe('CUJ: Creator can Play/Preview Video: Play and pause and reset video' () => {})` (can't use a comma in the action, thus the rewrite to and extra _"and"_).

Reply:
@barklund PTAL at a rewrite, what more would you put into fixture.editor?

CUJ: I think we can cover that in separate PR (general cleanup), that particular CUJ says nothing about autoplay, and its tasks are spread over various files.